lib/safe-pg-migrations/plugins/idem_potent_statements.rb

```diff
@@ -26,6 +26,13 @@ def remove_index(table_name, options = {})
       SafePgMigrations.say("/!\\ Index '#{index_name}' not found on table '#{table_name}'. Skipping statement.", true)
     end
 
+    def add_foreign_key(from_table, to_table, **options)
+      return super unless foreign_key_exists?(from_table, to_table) ||
```

Comment:
Not convinced that we should handle both cases, as we already change the signature of the original method to be `add_foreign_key(from_table, to_table, **options)`. If we do that, we might want to bump the minimal compatible Rails version to be 6.

Plus it's buggy, if I do
```rb
add_foreign_key :messages, :users, :author_id
```
and the column `messages.user_id` already exists, but not the `messages.author_id` column, it looks like it will not add the foreign key.